netlify.toml

```diff
@@ -20,3 +20,9 @@
   to = "https://docs.meltano.com/guide/troubleshooting/:splat"
   status = 301
   force = true
+
+[[redirects]]
+  from = "https://docs.meltano.com/cloud/*"
```

Comment:
```suggestion
  from = "/cloud/*"
```

Reply:
Actually, I'm not sure if we want to redirect in the preview builds 🤔

Reply:
See for example https://deploy-preview-8609--meltano.netlify.app/cloud/connectors/

Reply:
We'll be deleting the Cloud docs anyway in https://github.com/meltano/meltano/pull/8610, so I don't think there's any harm in making the redirect apply to preview builds too